src/api/app/views/webui2/webui/project/_monitor_control.html.haml

```diff
@@ -1,6 +1,8 @@
 .row
   .col-md-12
-    - if activate_client_search
+    = form_tag({:controller => 'project', :action => 'monitor'}, :method => :get) do
```

Comment:
[nitpick] use new Ruby syntax here (Rubocop should complain too)

Reply:
Fixed